package.json

```diff
@@ -128,6 +128,14 @@
                     ],
                     "default": true,
                     "description": "Set it as `false` if you don't want `.map` file for compiled CSS. \nDefault is `true`"
+                },
+                "liveSassCompile.settings.autoprefix": {
+                    "type": [
+                        "string",
+                        "null"
+                    ],
```

Comment:
Do  you not think it could be an array ?

(I don't know much about the configuration of browerlist).

I got this from https://github.com/ai/browserslist
```
"browserslist": [
    "> 1%",
    "last 2 versions"
  ]
```

Reply:
Might be better as an array, yes! I'll fix it as soon as possible.